packages/node_modules/@ciscospark/widget-recents/src/collab.scss

```diff
@@ -4,3 +4,14 @@ $images-path: "~@collab-ui/core/images";
 $docs-images-path: '~@collab-ui/core/docs/assets';
 
 @import '~@collab-ui/core/scss/collab-ui';
+
+.cui--dark .cui-button {
```

Comment:
Please add some comments around why this is is here

Reply:
Found out I don't need these classes, so I've removed them.